tests/script/test_evaluate.py

```diff
@@ -399,6 +399,7 @@ async def test_scriptEvaluate_realm(websocket, context_id):
     } == result
 
 
+@pytest.mark.timeout(30)
```

Comment:
Let's not triple it if possible, let's double it and then see if we can do something if we need to increase it again.
```suggestion
@pytest.mark.timeout(20)
```